### ISSUE TITLE
Can't run python server for testing

### ISSUE THREAD

**Issue by sashakid** [facebook/SocketRocket#405]
Hi, can't run python server for testingThe file /Users/alexyakovlev/Downloads/SocketRocket-master/pages/results/index.html does not exist.
make: *** [test] Error 1

**Comment by nlutsenko** [comment]
Hey @sashakid,
Try connecting to `localhost:9000` instead of `0.0.0.0`.

We are in the process of revamping how tests are set up and being run, so everyone can run them regardless of their local environment. Also, you can try patching #403 locally, so it's runnable.

I wouldn't recommend relying on `Makefile` to set up anything at the current moment.

**Comment by sashakid** [comment]
@nlutsenko nothing changes, please, check my screenshot:

![screenshot](https://cloud.githubusercontent.com/assets/5786373/15920832/e68a24c0-2e25-11e6-8bdd-be9948b39c01.jpg)

**Comment by sashakid** [comment]
Just didn't noticed in python code right URL 
```    
    application = tornado.web.Application([
        (r"/chat", ChatHandler),
        (r"/(.*)", tornado.web.StaticFileHandler, {"path": args.static_path, "default_filename":'index.html'}),
    ])
```
So the right URL to connect from iOS client is `[NSURL URLWithString:@"ws://localhost:9000/chat"]`

**Comment by nlutsenko** [comment]
(－‸ლ) That'll do it.
Did you get it working? If yes - I would gladly review the PR that updates the iOS client to the proper URL.

**Comment by sashakid** [comment]
Yes, it's working. I think you should mention it in your tutorial because it's not obvious for beginners or whom is not strong in networking like me :-) 

So, if you decide to test iOS client in simulator you should connect to `"ws://localhost:9000/chat"` and check the same URL in python server code in this line:39 `parser.add_argument('-i', '--listen-interface', help='interface to listen on. [default: %(default)s]', default='localhost', metavar='IFACE')`. 

But if you want to test on real device, you shoul change URL to your local network IP both in iOS client code and python server code, for example `"ws://YOUR_LOCAL_IP:9000/chat"`. 

For now it's `'0.0.0.0'`in python code, don't know what's the difference between `localhost` and `0.0.0.0` :-)
